File: spring-reactive-java/src/main/java/com/example/demo/DemoApplication.java
package com.example.demo;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@SpringBootApplication
public class DemoApplication {

    public static void main(String[] args) throws InterruptedException {
        Thread.sleep(10_000);
        SpringApplication.run(DemoApplication.class, args);
    }

}

@RestController
class Controller {

    @GetMapping("/hello")
    public String hello() {
        return "Hello World";
    }
}

File: spring-reactive-kt/src/main/kotlin/com/example/demo/DemoApplication.kt
package com.example.demo

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@SpringBootApplication
class DemoApplication

fun main(args: Array<String>) {
	Thread.sleep(10_000)
	runApplication<DemoApplication>(*args)
}


@RestController
class Controller {

	@GetMapping("/hello")
	fun hello() = "Hello  World"
}


